packages/lib/shared/components/poppover/BalPopover.tsx

```diff
@@ -0,0 +1,20 @@
+import { Popover, PopoverContent, PopoverTrigger, Text } from '@chakra-ui/react'
+import { PropsWithChildren } from 'react'
+
+type BalPopoverProps = {
+  text: string
+}
+
+export function BalPopover({ children, text }: PropsWithChildren<BalPopoverProps>) {
```

Comment:
Not used yet but we could iterate this new component to clear Popover related code